src/SchemaGenerator/SchemaClassGenerator.php

```diff
@@ -212,11 +220,16 @@ protected function generateInputObject(string $objectName): bool
      */
     protected function generateEnumObject(string $objectName): bool
     {
+        if (isset($this->generatedObjects[$objectName])) {
```

Comment:
I generally like the direction of the reworking you did in this file, I think that this structure is cleaner. That being said, do you have a reason why you're using `isset()` instead of `array_key_exists()`?

While `array_key_exists()` doesn't provide performance or safety benefits (at least ones that I know of), I find it more semantically correct to use in such use cases. I'm open to having `isset()` if you have a specific reason for making the switch.

Reply:
isset is slightly faster but you're right that array_key_exists is better for readability. And the small speed improvement is of no benefit in this situation.